README.md

```diff
@@ -91,6 +91,14 @@ Create a EFS storage on AWS and make sure it is in the same VPC as your cluster.
 export NFSSERVER=fs-d660549f.efs.us-east-1.amazonaws.com
 ```
 
+***Note!**
```

Comment:
I think this is missing an asterisk which is causing the rest to be bold?

Reply:
Ups, thanks, fixed it.